Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Magnataur CoA Update & 4 new CoAs

Added 10 Magnataur flags.

# How to test:
Start game.
Look at Magnataur CoAs.
Check if the CoAs are randomly generated for Magnataurs.
